packages/apollo-cli/src/config.ts

```diff
@@ -19,6 +19,7 @@ export interface SchemaDependency {
   engineKey?: string;
   extends?: string;
   clientSide?: boolean;
+  skipsSSLValidation?: boolean;
```

Comment:
I think this should probably be part of [`EndpointConfig`](
https://github.com/apollographql/apollo-cli#endpoint-configuration), similar to endpoint-specific headers.

Reply:
I think this is a great suggestion and actually I thought about implementing this way on the first place. 👍